build.gradle

```diff
@@ -16,7 +16,7 @@ plugins {
 
 ext {
   currentDate = new Date().format("d. MMM yyyy")
-  project.version = "0.1.0 DRAFT ($currentDate)"
+  project.version = project.file("./document.version").text
```

Comment:
Should probably add a fallback here if that file doesn't exist.

Reply:
No. I want the build script to fail hard if it is not there, because it is a mandatory file.